data/json/construction.json

```diff
@@ -7831,5 +7831,17 @@
     "pre_special": "check_empty",
     "post_special": "done_appliance",
     "activity_level": "LIGHT_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_obstacle_rock",
+    "group": "obstacle_rock",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "survival", 1 ] ],
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "time": "5 m",
+    "components": [ [ [ "rock_large", 6 ] ] ],
```

Comment:
I feel like this should be more than 36L of rocks, especially given a movecost of 4.

Reply:
How many would you use?

Reply:
24 or so - assuming they're perfect spheres, should be about enough to cover the whole ground in rocks.

Reply:
I decided for 22, to make it uneven enough to slow people down. Also, me no good at math.